recipes/neddy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "neddy" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 414ca082adde63a513eaea7ae5f789bf7903abe6aa13a428092748b00c212041
+
+build:
+  number: 0
+  entry_points:
+    - neddy=neddy.cl_utils:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+  run:
+    - fundamentals
+    - python >=3.9
+    - pyyaml
+    - astrocalc
+    - eventlet
+    - numpy
+    - unicodecsv
+
+test:
```

Comment:
I recommend adding a `pip check` test:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L61-L64

as well as adding a `commands` test for the entry point:
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#test-commands